apps/vpnmanager/sentry.server.config.ts

```diff
@@ -1,8 +1,11 @@
 import * as Sentry from "@sentry/nextjs";
 
-Sentry.init({
-  dsn: process.env.SENTRY_DSN,
-  environment: process.env.SENTRY_ENV,
-  tracesSampleRate: 1,
-  debug: false,
-});
+export function initSentry() {
+  Sentry.init({
+    dsn: process.env.SENTRY_DSN,
+    environment: process.env.SENTRY_ENV ?? "local",
+    tracesSampleRate: 1,
+  });
+}
+
+initSentry();
```

Comment:
Why not leave `Sentry.init` as is and define `initSentry` as:

```suggestion
export function initSentry() {
  Sentry.init();
}
```
?